web/connectionHub.go

```diff
@@ -22,49 +22,72 @@ var newRoom = make(chan models.RoomCreation)
 
 var Configs *Global.Configuration
 
+// Handles Actions
+
+const (
+	ActionRegistered = "REGISTERED"
+)
+
+// Message Types
+
+const (
+	InitialConnection = "initial-connection"
+	UserJoined        = "user-joined"
+)
+
 func handleRegistration(connection *models.UserConnection) {
```

Comment:
I think it was a good idea to change `entry` to "newRoom" in `handleNewRoom`, so similarly, it might make sense to change `entry` to `room` or something similar here as well.